src/PID.cc

```diff
@@ -136,7 +136,7 @@ double PID::Update(const double _error,
                    const std::chrono::duration<double> &_dt)
 {
   if (_dt == std::chrono::duration<double>(0) ||
-      ignition::math::isnan(_error) || std::isinf(_error))
+      isnan(_error) || std::isinf(_error))
```

Comment:
This one may be worth differentiating, so people reading the code know that it isn't `std::isnan`

Reply:
```suggestion
      ignition::math::isnan(_error) || std::isinf(_error))
```